tests/e2e/tests/smoke/browser/edit-key-name.e2e.ts

```diff
@@ -14,7 +15,11 @@ fixture `Edit Key names verification`
     .meta({ type: 'smoke' })
     .page(commonUrl)
     .beforeEach(async () => {
-        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        if(await addRedisDatabasePage.addDatabaseButton.visible) {
+            await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        } else {
+            await acceptLicenseAndConnectToRedisStack();
+        }
```

Comment:
I see it updated over many tests. Consider make a one main function that hold these statements.

Reply:
done